src/main/java/bowling/domain/Players.java

```diff
@@ -0,0 +1,35 @@
+package bowling.domain;
+
+import bowling.exception.InvalidPlayersException;
+
+import java.util.List;
+
+public class Players {
+    private final List<String> players;
```

Comment:
```suggestion
    private final List<Player> players;
```

와 같이 조금 더 작은 단위로 객체를 분리해볼 수 있겠네요 ㅎㅎ

Reply:
하나 더 작은 단위로 분리 가능하네요 ㅎㅎ 해당 부분도 적용 했습니다!